code/datums/components/bullet_holes.dm

```diff
@@ -46,32 +48,35 @@ TYPEINFO(/datum/component/bullet_holes)
 	. = ..()
 
 
-/datum/component/bullet_holes/proc/handle_impact(rendering_on, obj/projectile/shot)
+/datum/component/bullet_holes/proc/handle_impact(rendering_on, obj/projectile/shot, impact_x, impact_y)
 	var/datum/projectile/shotdata = shot.proj_data
+	// Shots coming from the south hit the top part of the wall. Perspective means we do not spawn a decal
+	if (shot.dir == SOUTH)
+		return
```

Comment:
Here, for some reason GitHub didn't put in the right place

Reply:
Sorry, the comment makes it a bit confusing. Ill rewrite it to "shots GOING to the south".
Since the shot is going down it'll hit the top part of the wall and due to perspective we do not draw a decal